<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1033</width>
    <height>968</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTextBrowser" name="info_1">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>771</width>
      <height>61</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>连接确认信息</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_Connect">
    <property name="geometry">
     <rect>
      <x>830</x>
      <y>30</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>连接测试仪</string>
    </property>
   </widget>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>981</width>
      <height>211</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <property name="lineWidth">
     <number>1</number>
    </property>
    <property name="midLineWidth">
     <number>0</number>
    </property>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>440</x>
       <y>70</y>
       <width>56</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Aharoni</family>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>CrossCursor</cursorShape>
     </property>
     <property name="text">
      <string>掩码</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>板卡</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>20</y>
       <width>211</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>440</x>
       <y>20</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>端口号</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_ok">
     <property name="geometry">
      <rect>
       <x>820</x>
       <y>70</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>确认</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="line_mask">
     <property name="geometry">
      <rect>
       <x>520</x>
       <y>70</y>
       <width>201</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="geometry">
      <rect>
       <x>-1</x>
       <y>-1</y>
       <width>981</width>
       <height>221</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="title">
      <string>待测设备连接信息</string>
     </property>
     <widget class="QLineEdit" name="line_port">
      <property name="geometry">
       <rect>
        <x>520</x>
        <y>20</y>
        <width>201</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="line_ip">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>70</y>
        <width>211</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>70</y>
        <width>81</width>
        <height>30</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Aharoni</family>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>CrossCursor</cursorShape>
      </property>
      <property name="text">
       <string>设备地址</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="info_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>110</y>
        <width>941</width>
        <height>91</height>
       </rect>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
    <zorder>groupBox_2</zorder>
    <zorder>label_4</zorder>
    <zorder>label_10</zorder>
    <zorder>comboBox</zorder>
    <zorder>label_11</zorder>
    <zorder>btn_ok</zorder>
    <zorder>line_mask</zorder>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>310</y>
      <width>981</width>
      <height>591</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="title">
     <string>测试项选择</string>
    </property>
    <widget class="QTabWidget" name="tabWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>831</width>
       <height>231</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <property name="usesScrollButtons">
      <bool>false</bool>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>协议类型</string>
      </attribute>
      <widget class="QListWidget" name="listWidget">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>831</width>
         <height>201</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>9</pointsize>
        </font>
       </property>
       <item>
        <property name="text">
         <string>(01) ModbusTcp Master</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>(02) ModbusTcp Slave</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>(03) ModbusRTU Master</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>(04) ModbusRTU Slave</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>(05) CAN</string>
        </property>
       </item>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>测试类</string>
      </attribute>
      <widget class="QTextBrowser" name="typeInfo">
       <property name="geometry">
        <rect>
         <x>530</x>
         <y>0</y>
         <width>281</width>
         <height>201</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
      </widget>
      <widget class="QListWidget" name="listWidget_2">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>531</width>
         <height>211</height>
        </rect>
       </property>
      </widget>
      <zorder>listWidget_2</zorder>
      <zorder>typeInfo</zorder>
     </widget>
    </widget>
    <widget class="QPushButton" name="btn_loadtest">
     <property name="geometry">
      <rect>
       <x>870</x>
       <y>60</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>测试加载</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_checktest">
     <property name="geometry">
      <rect>
       <x>870</x>
       <y>100</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>测试检查</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_starttest">
     <property name="geometry">
      <rect>
       <x>870</x>
       <y>140</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>测试执行</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_pausetest">
     <property name="geometry">
      <rect>
       <x>870</x>
       <y>180</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>测试暂停</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_endtest">
     <property name="geometry">
      <rect>
       <x>870</x>
       <y>220</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>测试结束</string>
     </property>
    </widget>
    <widget class="QProgressBar" name="prograssBar">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>280</y>
       <width>881</width>
       <height>23</height>
      </rect>
     </property>
     <property name="value">
      <number>24</number>
     </property>
     <property name="textVisible">
      <bool>true</bool>
     </property>
     <property name="textDirection">
      <enum>QProgressBar::TopToBottom</enum>
     </property>
    </widget>
    <widget class="QTextBrowser" name="text_info">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>330</y>
       <width>931</width>
       <height>221</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
    </widget>
    <zorder>tabWidget</zorder>
    <zorder>prograssBar</zorder>
    <zorder>text_info</zorder>
    <zorder>btn_pausetest</zorder>
    <zorder>btn_checktest</zorder>
    <zorder>btn_loadtest</zorder>
    <zorder>btn_endtest</zorder>
    <zorder>btn_starttest</zorder>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1033</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>测试平台</string>
    </property>
    <addaction name="actionConnect"/>
    <addaction name="actionDisconnect"/>
    <addaction name="actionConfigure"/>
   </widget>
   <widget class="QMenu" name="menuLinkEasy">
    <property name="title">
     <string>LinkEasy</string>
    </property>
   </widget>
   <addaction name="menu"/>
   <addaction name="menuLinkEasy"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionConnect">
   <property name="text">
    <string>Connect(&amp;C)</string>
   </property>
  </action>
  <action name="actionDisconnect">
   <property name="text">
    <string>Disconnect(&amp;D)</string>
   </property>
  </action>
  <action name="actionConfigure">
   <property name="text">
    <string>Configure</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
